tsl/test/expected/dist_compression.out

```diff
@@ -1388,3 +1388,87 @@ ORDER BY 1;
  c2
 (1 row)
 
+-- test ADD COLUMN IF NOT EXISTS on a distributed hypertable
+CREATE TABLE metric (time TIMESTAMPTZ NOT NULL, val FLOAT8 NOT NULL, dev_id INT4 NOT NULL);
+ 
+SELECT create_distributed_hypertable('metric', 'time');
+ create_distributed_hypertable 
+-------------------------------
+ (10,public,metric,t)
+(1 row)
+
+ALTER TABLE metric SET (                                     
+timescaledb.compress,
+timescaledb.compress_segmentby = 'dev_id',
+timescaledb.compress_orderby = 'time DESC'
+);
+ 
+INSERT INTO metric(time, val, dev_id)
+SELECT s.*, 3.14+1, 1
+FROM generate_series('2021-07-01 00:00:00'::timestamp,
+                    '2021-08-17 00:02:00'::timestamp, '30 s'::interval) s;
+ 
+SELECT compress_chunk(show_chunks('metric'));
```

Comment:
should rewrite the query with order by (so that test output is consistent)
SELECT compress_chunk(ch) FROM show_chunks('metric') ch ORDER BY 1;